packages/wrangler/src/api/startDevWorker/ProxyController.ts

```diff
@@ -67,8 +67,8 @@ export class ProxyController extends EventEmitter {
 			workers: [
 				{
 					name: "ProxyWorker",
-					// `url_standard` required to parse IPv6 hostnames correctly
```

Comment:
Without this do we still work with IPv6 hostnames?

Reply:
Oh right. So the fix is actually adding a compatibility date, not ther removal of the compatibility flag?